test/OpenTelemetry.Collector.Dependencies.Tests/HttpClientTests.cs

```diff
@@ -100,11 +100,13 @@ public async Task HttpOutCallsAreCollectedSuccessfullyAsync(HttpOutTestCase tc)
 
             var startEndHandler = new Mock<IStartEndHandler>();
             var tracer = new Tracer(startEndHandler.Object, new TraceConfig());
```

Comment:
is the `tracer` instance still needed?

Reply:
It is returned be the mocked `TracerFactory`